src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -71,9 +74,37 @@ public void AddSentryFramework()
             _project.SetBuildProperty(unityFrameworkTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(inherited)");
             _project.AddBuildProperty(unityFrameworkTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(PROJECT_DIR)/Frameworks/");
 
+            _project.AddBuildProperty(unityFrameworkTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(PROJECT_DIR)/Frameworks/");
+            _project.AddBuildProperty(unityFrameworkTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(PROJECT_DIR)/Frameworks/");
+
+            _project.SetBuildProperty(mainTargetGuid, "DEBUG_INFORMATION_FORMAT", "dwarf-with-dsym");
+            _project.SetBuildProperty(unityFrameworkTargetGuid, "DEBUG_INFORMATION_FORMAT", "dwarf-with-dsym");
+
             _project.AddBuildProperty(mainTargetGuid, "OTHER_LDFLAGS", "-ObjC");
         }
 
+        public void AddBuildPhaseSymbolUpload(IDiagnosticLogger? logger)
+        {
+            if (MainTargetContainsSymbolUploadBuildPhase())
+            {
+                logger?.LogDebug("Build phase '{0}' already added.", SymbolUploadPhaseName);
+                return;
+            }
+
+            var mainTargetGuid = _project.GetUnityMainTargetGuid();
+            _project.AddShellScriptBuildPhase(mainTargetGuid,
+                SymbolUploadPhaseName,
+                "/bin/sh",
+                @"export SENTRY_PROPERTIES=sentry.properties
+if [[ ""$BITCODE_GENERATION_MODE"" == ""marker"" ]] ; then
+    ERROR=$(./sentry-cli-Darwin-universal upload-dif $BUILT_PRODUCTS_DIR > ./sentry-symbols-upload.log 2>&1 &)
```

Comment:
We'll always make sure `sentry-cli-Darwin-universal` is executing directory?

Reply:
Yes. I replaced the magic string with the const from SentryCli. 
We want to make sure that we add the Darwin-universal because I'm assuming that the Xcode project only gets built on a mac.